index.html

```diff
@@ -229,38 +229,71 @@ <h3>
             <dt id="data-integrity-1" class="spec">Verifiable Credential Data Integrity 1.0</dt>
             <dd>
               <p>
-This specification defines how to express proofs of integrity, such as digital
-signatures or proofs of existence, for bounded documents, such as verifiable
-credentials. Concrete serializations will be provided based on <a
-href="https://w3c.github.io/vc-data-model/#json-web-token">VC-JWT</a>, the <a
-href="https://w3c-ccg.github.io/lds-ed25519-2020/">Ed25519 Cryptosuite</a>, 
-the <a href="https://w3c-ccg.github.io/lds-ecdsa-secp256k1-2019/">Secp256k1
-Cryptosuite</a>, and for <a href="https://github.com/w3c-ccg/lds-jws2020/">
-JSON Web Signature 2020</a>. Concrete serializations based on the <a
-href="https://w3c-ccg.github.io/ldp-bbs2020/">BBS+ Cryptosuite</a> might be provided, if an <a
-href="https://identity.foundation/bbs-signature/">IETF BBS+ RFC</a> is published
-in time. Concrete serializations for VC-JWP might be provided, if an <a
-href="https://json-web-proofs.github.io/json-web-proofs/">IETF JWP RFC</a> is
-published in time. 
-The final output specifications might not map directly to
-the list of input drafts. Other output serializations covering cryptographic
-primitives published in IETF RFCs not mentioned in this paragraph might be
-produced.
+This family of specifications defines how to express proofs of integrity for
+verifiable credentials using a number of concrete serializations for each of the
+defined syntaxes. The specific set of concrete serializations included will be
+determined by the Working Group. The following are a non-exhaustive selection
+of expected input documents:
```

Comment:
I would like to make it clearer what constitutes an `input document`. What do we do with them? If we are to define how containers/cryptosuites defined in those documents can be used with vc-data-model, I would want these input documents to be "published standards", and I think language below would be clearer:
```suggestion
of expected input documents that are published standards:
```

Reply:
In W3C, an "input document" is something that isn't a published standard yet. It's just a place from which the WG can start work, and is typically a document that has been incubated in a group that has an IPR policy that is compatible with W3C (such as a W3C community group or IETF group and/or Independent Draft).

Reply:
Thank you for clarifying - is there a W3C glossary somewhere?

Reply:
> Thank you for clarifying - is there a W3C glossary somewhere?

Unfortunately, the [initial W3C glossary work](https://www.w3.org/Glossary), and [subsequent glossary work](https://www.w3.org/2003/glossary/) was abandoned in 2010.

Since this is charter stuff, the best reference is the [W3C Process](https://www.w3.org/2021/Process-20211102/) document. Even that [barely mentions "input"](https://www.w3.org/2021/Process-20211102/#SubmissionScope) in the sense that it's being used here (mostly because the term only comes up at chartering time and the question only exists for a very brief period of a few weeks before people stop talking about "input documents").

Hope that helps.